package.json

```diff
@@ -6,8 +6,8 @@
   "module": "src/plugin.js",
   "scripts": {
     "test": "npm run lint && jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "test-lib": "npm run lint && jest tests/lib --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
-    "test-components": "npm run lint && jest tests/vue --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
+    "test-lib": "npm run lint && jest tests/lib --coverage",
+    "test-components": "npm run lint && jest tests/vue --coverage",
```

Comment:
It is for passing travis tests - otherwise it throws an error after passing successfull tests.